gax/src/test/java/com/google/api/gax/batching/BatcherImplTest.java

```diff
@@ -69,8 +69,8 @@
   private final LabeledIntList labeledIntList = new LabeledIntList("Default");
   private final BatchingSettings batchingSettings =
       BatchingSettings.newBuilder()
+          .setElementCountThreshold(1000L)
           .setRequestByteThreshold(1000L)
-          .setElementCountThreshold(1000)
           .setDelayThreshold(Duration.ofSeconds(1))
```

Comment:
Mybe it is a wrong place to looks at it, but PR description says:
"Defaulting to 1**ms** in case delayThreshold value is unset."
Here is seems like it defaults to 1 sec. Is something wrong here or am I just looking in a wrong place?

Reply:
I think rahul forgot to update the description of the PR :)

Reply:
I am really sorry, Have updated the description now.
We are not scheduling any auto flush if the `delayThreshold` is unset.